### PR TITLE
chore: Add JP translation for README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by poppoerika**
Added JP translation for both `README` and `CONTRIBUTING`

@danielamiao - It seems like the example has table of contents but since this README is short I'm not sure if it's needed. 
I skipped it for now.

### CODE REVIEW

File: CONTRIBUTING.ja.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+### 開始方法
+
+1. git submodule init
```
Comment (danielamiao) [inline CONTRIBUTING.ja.md:3]: This line and below all have numbers at the beginning, you should delete it so users can copy the entire code block to terminal without having to delete the numbers

File: README.ja.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+## 必要条件
```
Comment (danielamiao) [inline README.ja.md:1]: I like the sample README had a line at the top stating other languages are available, linking to each README version. Could we add it here too? That way, if a Japanese user lands on the English README by accident they can easily click on the link to find the Japanese README

Comment (danielamiao) [reply README.ja.md:1]: Like this line 

Read this in other languages: [English](https://github.com/tiimgreen/github-cheat-sheet/blob/master/README.md), [한국어](https://github.com/tiimgreen/github-cheat-sheet/blob/master/README.ko.md), [日本語](https://github.com/tiimgreen/github-cheat-sheet/blob/master/README.ja.md), [简体中文](https://github.com/tiimgreen/github-cheat-sheet/blob/master/README.zh-cn.md), [正體中文](https://github.com/tiimgreen/github-cheat-sheet/blob/master/README.zh-tw.md).

**Review by danielamiao — COMMENTED** [review]
Agree table of content is not needed, a few other comments!

File: README.ja.md (hunk 2 above)
Comment (poppoerika) [reply README.ja.md:1]: Added to the original ‘README.md`! 
I can add the same for the Japanese README as well.

**Review by danielamiao — APPROVED** [review]

**Pull request merged**